File: nom-ui/src/app/household/models/household-create-response.model.ts
// File: nom-ui/src/app/household/models/household-create-response.model.ts

import { IHouseholdCreateResponseModel } from './household-create-response.model.interface';

export class HouseholdCreateResponseModel implements IHouseholdCreateResponseModel {
    id = 0;
    name = '';
    description?: string;
    createdDate: Date = new Date();
    inviteToken = '';

    constructor(data?: Partial<IHouseholdCreateResponseModel>) {
        if (data) {
            Object.assign(this, data);
        }
    }
} 

File: nom-api/Nom.Orch/Models/Household/HouseholdCreateResponseModel.cs
// File: Nom.Orch/Models/Household/HouseholdCreateResponseModel.cs

namespace Nom.Orch.Models.Household
{
    public class HouseholdCreateResponseModel
    {
        public long Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public long GroupId { get; set; }
        public DateTime CreatedDate { get; set; }
    }
} 